docs/pages/setup/reference/license.mdx

```diff
@@ -21,9 +25,16 @@ The `license_file` path can be either absolute or relative to the configured
 `data_dir`. If the license file path is not set, Teleport will look for the
 `license.pem` file in the configured `data_dir`.
 
+The Teleport l
+
 <Admonition
   type="tip"
   title="NOTE"
 >
-  Only Auth Servers require the license. Proxies and Nodes that do not also have Auth role enabled do not need the license.
+  Only Auth Servers require the license. Proxies and Nodes that do not also have Auth role enabled do not need the license.  
+    
+    Attempts to use unlicenced products will result in a error message and users will be unable to use those products.  Here is an example message if attempting to use database access and not included in the license.
+```bash
```

Comment:
The coloration is a bit off here. Maybe use `code` instead of `bash`? Or is there a `text` type?